### PR TITLE
IGVF-143-rename-organoid-in-vitro

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/schemas/changelogs/differentiated_cell.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## Changelog for differentiated_cell.json
+* renamed in_vitro.json to differentiated_cell.json
```
Comment (meenakshikagda) [inline src/igvfd/schemas/changelogs/differentiated_cell.md:2]: Try to follow this example: https://github.com/IGVF-DACC/igvfd/blob/IGVF-128-rm-parenthesis-temp-schema/src/igvfd/schemas/changelogs/treatment.md

Start with the title "Minor revision..." And write it in a sentence structure (caps your first word in the. sentence)

File: src/igvfd/schemas/changelogs/differentiated_tissue.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+## Changelog for differentiated_tissue.json
+* renamed organoid.json to differentiated_tissue.json
```
Comment (meenakshikagda) [inline src/igvfd/schemas/changelogs/differentiated_tissue.md:2]: Same as above. See https://github.com/IGVF-DACC/igvfd/blob/IGVF-128-rm-parenthesis-temp-schema/src/igvfd/schemas/changelogs/treatment.md for example.

File: src/igvfd/schemas/changelogs/differentiated_cell.md (hunk 1 above)
Comment (meenakshikagda) [reply src/igvfd/schemas/changelogs/differentiated_cell.md:2]: You may use something on these lines: the original profile named "in_vitro" is now renamed to be "differentiated_cell"

**Review by meenakshikagda — DISMISSED** [review]

File: src/igvfd/schemas/changelogs/differentiated_cell.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+## Changelog for differentiated_cell.json
+
+### Minor changes since version 1
+* Renmaed the original profile named in_vitro to differentiated_cell.
```
Comment (jenjou) [inline src/igvfd/schemas/changelogs/differentiated_cell.md:4]: There's a minor typo here

File: src/igvfd/schemas/changelogs/differentiated_tissue.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+## Changelog for differentiated_tissue.json
+
+### Minor changes since version 1
+* Renmaed the original profile named organoid to differentiated_tissue.
```
Comment (jenjou) [inline src/igvfd/schemas/changelogs/differentiated_tissue.md:4]: Same typo here as well.

**Review by meenakshikagda — APPROVED** [review]

**Pull request merged**